### ISSUE TITLE
Unable to find co-ords to set window open location

### ISSUE THREAD

**Issue by cryptominerau** [aasmal97/Window-Actions#19]
When setting up the move to different screen location, the mover does shift the window to the other screen, then when the resize/location command runs it snaps back to the original screen as i dont know a way to specify the correct x,y co-ords for the screens. How do i obtain these values?

**Comment by aasmal97** [comment]
Absolutely, when I have some time, I'll work on improving this functionality to make more sense for the end user. 

However, x and y coordinates work by using negative and positive values relative to your PRIMARY monitor.

![RDT_20241207_1243327249162118962631048](https://github.com/user-attachments/assets/216f9eca-e4a1-4030-b5d1-c7f578ebd49b)

In the following layout, if I want the window to be on the top right second monitor, the coordinates would be 0, 3840, because my first and PRIMARY monitor is 3840x2160 because it's a 4k monitor. 

If your primary monitor is in the center, and you want to move it to a monitor on the left, you'd have to use a negative value and the resolution of that monitor.

**Comment by cryptominerau** [comment]
How would i calculate with my screen set up? Screen 1 is the primary, 1+2 are 4k, 3 is a 1440 ultrawide.

![image](https://github.com/user-attachments/assets/9b5287a4-2826-4b92-8a27-2e0496238e43)

From the apps on the streamdeck store i think your one is by far the best this is currently the only issue. Is there a way you could add it so you can pre-place the window, and then the app would just pull its current size and placement and save that? Allowing you to set the screen up, and basically snapshot the wanted windows in that size and place?

**Comment by cryptominerau** [comment]
(misclicked the close button sorry!)

**Comment by cryptominerau** [comment]
So i figured out a work around for the moment. Using "Windows Mover [BarRaider] to use the "select current position function" an a random app thats been placed where you want it will give you the X Y and size L W and then plug that into this tool, worked a treat!
